saleor_app_framework/app_framework/core/manifest.py

```diff
@@ -1,17 +1,18 @@
 import json
 import os
 
-from .conf import settings
+from .conf import get_settings
 
 MANIFEST = None
 
 
 def get_app_manifest():
     global MANIFEST
+    settings = get_settings()
     if MANIFEST is not None:
         return MANIFEST
-    if not os.path.exists(settings.MANIFEST_PATH):
+    if not os.path.exists(settings.manifest_path):
```

Comment:
This can now be `settings.manifest_path.exists()`

Reply:
I believe we can remove this check. As pydantic will do the job for us :)

Reply:
Thanks for catching this

Reply:
Since it's not optional, right? :+1: